packages/core-app-api/src/app/AppThemeProvider.tsx

```diff
@@ -90,20 +88,9 @@ export function AppThemeProvider({ children }: PropsWithChildren<{}>) {
     throw new Error('App has no themes');
   }
 
-  if (appTheme.Provider) {
-    return <appTheme.Provider children={children} />;
+  if (!appTheme.Provider) {
```

Comment:
I wanted to bring to your attention this change for review. Since _theme_ is no longer an option, I have removed the logic that instantiates a _ThemeProvider_ from _theme_. It now throws an error if _Provider_ is not included. Let me know if this is/isn't the right approach!

Reply:
Tbh I think it should be alright to remove this check and error. Type checking *should* always find this issue anyway so it's arguably a bit too defensive. 👍 removing the logic though, that's part of what was deprecated

Reply:
Hm, it looks like without the `!appTheme.Provider` check, a `tsc` error is thrown stating that, "JSX element type 'appTheme.Provider' does not have any construct or call signatures." Is that because the `Provider` is optional, and can be null?

Reply:
Commented on it below too, but the `Provider` should be made required now. The previous warning message was there to warn when not defining a `Provider`